### PR TITLE
fix: use import attribute in release-post script

### PR DESCRIPTION
**Pull request by targos**
## Description

Node.js 22.0.0 removed support for import assertions.
The new syntax works in recent versions of Node.js 20 and 18.

## Validation

Preparing the release post for v22.1.0 with it.

### Check List

- [ ] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [ ] I have run `npm run format` to ensure the code follows the style guide.
- [ ] I have run `npm run test` to check if all tests are passing.
- [ ] I have run `npx turbo build` to check if the website builds without errors.
- [ ] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

**Review by AugustinMauroy — APPROVED** [review]
LGTM !

**Review by bmuenzenmeyer — APPROVED** [review]

**Review by canerakdas — APPROVED** [review]

File: scripts/release-post/index.mjs [hunk 1]
```diff
@@ -28,7 +28,7 @@ import handlebars from 'handlebars';
 import { format } from 'prettier';
 
 import { downloadsTable } from './downloadsTable.mjs';
-import prettierConfig from '../../.prettierrc.json' assert { type: 'json' };
+import prettierConfig from '../../.prettierrc.json' with { type: 'json' };
```
Comment (ovflowd) [inline scripts/release-post/index.mjs:31]: We have import assertions in other places of the codebase, do you mind replacing those in this PR too?

Comment (targos) [reply scripts/release-post/index.mjs:31]: Sorry but I'm not interested in doing more than this.

**Review by ovflowd — APPROVED** [review]
LGTM

File: scripts/release-post/index.mjs (hunk 1 above)
Comment (ovflowd) [reply scripts/release-post/index.mjs:31]: @nodejs/nodejs-website does someone have the bandwidth for that?

Comment (AugustinMauroy) [reply scripts/release-post/index.mjs:31]: on node 20 webpack's not happy but on 22 it's build correctly so let's do nothing for moment IMHO

**Review by aymen94 — APPROVED** [review]

**Pull request merged**